extensions/widgets/androidfield/androidfield.lcb

```diff
@@ -1769,10 +1769,6 @@ public handler SetScrollingEnabled(in pEnabled as Boolean)
 end handler
 
 public handler GetScrollingEnabled() returns Boolean
-   if mNativeObj is not nothing then
-      put TextView_getMovementMethod(mNativeObj) into mScrollingEnabled
-   end if
-
```

Comment:
you can still query the TextView object for this value - I assume the error was due to assigning an optional JObject value to a boolean. A working version of this would be `put (TextView_getMovementMethod(mNativeObj) is not nothing) into mScrollingEnabled`